ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTablePostgresql.scala

```diff
@@ -176,9 +176,11 @@ object EventsTablePostgresql extends EventsTable {
     val exerciseChildEventIds = "exerciseChildEventIds"
   }
 
-  private val insertStmt = {
+  private val conflictActionClause = if (idempotentEventInsertions) "on conflict do nothing" else ""
```

Comment:
This ignores all conflicts. Could you add a comment what potential conflicts there are, and why it's ok to ignore all of them? It's hard to look up what constraints and indices we have on the `participant_events` table.

Reply:
Done. Added a doc specifying the conflict clause and why is needed.